homeassistant/components/sensor/counter.py

```diff
@@ -0,0 +1,102 @@
+"""
+Support for keeping a simple counter.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.counter/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import DOMAIN, PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_INITIAL = 'initial'
+CONF_STEP = 'step'
+
+DEFAULT_NAME = 'Counter'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_INITIAL, default=0): vol.Coerce(int),
+    vol.Optional(CONF_STEP, default=1): vol.Coerce(int)
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the sensor platform."""
+    data = CounterData(hass, config)
+    sensor = CounterSensor(data, config.get(CONF_NAME))
+    add_devices([sensor])
+
+    def increment_counter(call=None):
+        """Increment counter and update sensor."""
+        data.increment()
+        sensor.update()
+
+    def decrement_counter(call=None):
+        """Decrement counter and update sensor."""
+        data.decrement()
+        sensor.update()
+
+    def reset_counter(call=None):
+        """Reset counter and update sensor."""
+        data.reset()
+        sensor.update()
+
+    hass.services.register(DOMAIN, 'increment_counter', increment_counter)
```

Comment:
With prefix, do you mean the DOMAIN part that should be changed, or should the service call itself by changed (to `counter_increment` then I guess)?